utils/ansible/ftest/inventory-sample.yml

```diff
@@ -3,11 +3,13 @@
 all:
   vars:
     daos_runtime_dir: /home/foo/daos
+    # daos_ofi_provider: ofi+tcp;ofi_rxm
```

Comment:
If you comment it out for now and plan to use it later, you might want to add some explanation.

Reply:
This comment present and example of an optional parameters.
if it is confusing, I can remove it or add something like this 
```suggestion
    # Optional parameter
    # daos_ofi_provider: ofi+tcp;ofi_rxm
```

Reply:
- [ ] Add default parameter explanation

Reply:
> * [x]  Add default parameter explanation

Fixed with commit 07b4777df